indigo/distribution.yaml

```diff
@@ -15161,6 +15161,16 @@ repositories:
       url: https://github.com/ethz-asl/ethzasl_xsens_driver.git
       version: master
     status: maintained
+  xsens_awinda:
+    doc:
+      type: git
+      url: https://github.com/Raffa87/xsense-awinda.git
+      version: master
+   source:
```

Comment:
Please review your indentation it appears to be breaking the parsing as seen in the CI failure.